frontend/src/components/VisaStatusOverviewList/OverviewContent.js

```diff
@@ -0,0 +1,243 @@
+import React, { useMemo, useState } from "react";
+import PropTypes from "prop-types";
+import { useSelector } from "react-redux";
+import { Row, Col, Button, Tooltip, Space, Collapse, Tag, Modal } from "antd";
+import { MailOutlined, QqOutlined, EllipsisOutlined, PlusOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+import { useScreenXS } from "../../hooks";
+import { makeNewestVisaStatusSelector } from "../../redux/selectors";
+import { findEmbassyAttributeByCode } from "../../utils/USEmbassy";
+import { overviewAttrProps, newestOverviewProps, overviewProps } from "./overviewPropTypes";
+import "./VisaStatusOverview.less";
+
+const { Panel } = Collapse;
+
+const QQTGSubsModalHook = (qq, tg) => {
+    const { t } = useTranslation();
+    const [isModalVisible, setIsModalVisible] = useState(false);
+    return [
+        <Modal
+            title={t("QQTGModalTitle")}
+            visible={isModalVisible}
+            onOk={() => setIsModalVisible(false)}
+            onCancel={() => setIsModalVisible(false)}
+        >
+            <p>
+                {t("QQTGModalContentTG")}
+                <a href={tg} target="_blank" rel="noreferrer">
+                    {tg}
+                </a>
+            </p>
+            <p>{t("QQTGModalContentQQDesc")}</p>
+            {qq.map((content, index) => (
+                <p>
+                    {t("QQTGModalContentQQ", { index: index + 1 })}
+                    {content}
+                </p>
+            ))}
+        </Modal>,
+        setIsModalVisible,
+    ];
+};
+
+const ContentActions = ({ children, embassyCode, onEmailSubsClick, onAddtionClick }) => {
+    const { t } = useTranslation();
+    const info = useSelector(state => state.metadata.qqTgInfo);
+    const embassyLst = useSelector(state => state.metadata.embassyLst);
+    const region = findEmbassyAttributeByCode("region", embassyCode, embassyLst);
+    const index = region === "DOMESTIC" ? "domestic" : "nonDomestic";
```

Comment:
I believe this can refactor into a `reselect` selector where eventually we will have something like

```js
const [qqInfo, tgInfo] = useSelector(state => qqtgInfoSelector(state));
```